beeline/middleware/awslambda/test_awslambda.py

```diff
@@ -173,10 +173,17 @@ def foo(event, context):
 
             self.assertEqual(foo(None, None), 1)
 
+            @awslambda.beeline_wrapper()
+            def foo(event, context):
```

Comment:
we might need to rename this to get the linter to pass the build

Reply:
Updated in 6d16c2bc93acbc3ace6a21a39407358ff183d670 (thanks @MikeGoldsmith!)

Sorry for the delay, I've been a bit distracted the past week or so for obvious reasons 🙃